spanner/tests/system/test_system.py

```diff
@@ -282,6 +283,18 @@ def test_create_database(self):
             for database in Config.INSTANCE.list_databases()]
         self.assertIn(temp_db_id, database_ids)
 
+    def test_create_database_not_found(self):
+        temp_db_id = 'temp_db' + unique_resource_id('_')
+        temp_db = Config.INSTANCE.database("bitemporal_pattern1", ddl_statements=[
+            """CREATE TABLE MyTable (
+                Id        String(36) NOT NULL,
+                Field1  STRING(36) NOT NULL
+            ) PRIMARY KEY (Id)""",
+            """CREATE INDEX IDX ON yTable (Field1)"""
```

Comment:
1. Did you mean `MyTable` rather than `yTable`
1. Please only create triple-quote strings at module level (you end up with unintended indents)
1. Please use single quotes for `bitemporal_pattern1`
1. What is a `bitemporal_pattern1`? Maybe use a "normal" name?

Reply:
ok, I just grabbed the case from vikas' example.

Reply:
@chemelnucfin I see that now. It's up to us to translate feedback from issues into the codebase (and that usually means changing some parts).